libs/labelbox/src/labelbox/schema/task.py

```diff
@@ -49,6 +55,13 @@ class Task(DbObject):
     created_by = Relationship.ToOne("User", False, "created_by")
     organization = Relationship.ToOne("Organization")
 
+    def __eq__(self, task):
```

Comment:
To support the `DataUpsertTask` in the list `user.created_task()`
We really should refactor our tasks (sigh)